tests/monster_test.fbs

```diff
@@ -72,6 +72,14 @@ table Referrable {
   id:ulong(key, hash:"fnv1a_64");
 }
 
+enum instanceof: int {
```

Comment:
I wonder if this is best placed this in a separate file, as it kind of pollutes all the other languages that don't care about these keywords.

Reply:
I'll do that, although admittedly that's more because so many other languages barf on some of the keywords that it's hard to properly test the typescript case.... My original test case had keywords that were more common across languages (`class`, `new`, `Object`, etc.), but because other languages currently had errors, I ended up just working through the list of typescript keywords until I found ones that didn't trigger other languages' logic (and in the process was able to unearth the various other bugs I filed).